### PR TITLE
(fix) Offline actions component added

### PR DESCRIPTION
**Pull request by icrc-jofrancisco**
## Requirements
- [x] This PR has a title that briefly describes the work done including the ticket number. If there is a ticket, make sure your PR title includes a [conventional commit](https://o3-dev.docs.openmrs.org/#/getting_started/contributing?id=your-pr-title-should-indicate-the-type-of-change-it-is) label. See existing PR titles for inspiration.

#### For changes to apps
- [x]  My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide) and **design documentation**.

#### If applicable
- [ ] My work includes tests or is validated by existing tests.
- [ ] I have updated the [esm-framework mock](https://github.com/openmrs/openmrs-esm-core/blob/main/packages/framework/esm-framework/mock.tsx) to reflect any API changes I have made.

## Summary

This pull request addresses the accidental removal of the offline actions card that occurred during the core v5 migration. We are taking the necessary steps to reinstate the offline actions card to ensure its continued availability in our application.
## Screenshots

image

## Other

CC: @ibacher 
Thanks

### CODE REVIEW

File: packages/apps/esm-offline-tools-app/src/offline-actions/offline-actions-table.component.tsx [hunk 1]
```diff
@@ -261,7 +261,7 @@ function PatientLink({ patientUuid, patientName }) {
     <Link
       onClick={() =>
         navigate({
-          to: `${window.getOpenmrsSpaBase()}patient/${patientUuid}/chart`,
+          to: `${window.getOpenmrsSpaBase()}patient/${patientUuid}/chart/Patient Summary`,
```
Comment (vasharma05) [inline packages/apps/esm-offline-tools-app/src/offline-actions/offline-actions-table.component.tsx:264]: We can let it be to `/chart`, since the forwarding path from '/chart' is customizable.
Please revert this change.
Thanks!

File: packages/apps/esm-offline-tools-app/src/offline-patients/patient-name-table-cell.component.tsx [hunk 2]
```diff
@@ -23,7 +23,9 @@ const PatientNameTableCell: React.FC<PatientNameTableCellProps> = ({
       <Link
         onClick={() =>
           navigate({
-            to: `${window.getOpenmrsSpaBase()}patient/${patient.id}/chart`,
+            to: `${window.getOpenmrsSpaBase()}patient/${
+              patient.id
+            }/chart/Patient Summary`,
```
Comment (vasharma05) [inline packages/apps/esm-offline-tools-app/src/offline-patients/patient-name-table-cell.component.tsx:28]: /chart please.
Thanks!

File: packages/apps/esm-offline-tools-app/src/index.ts [hunk 3]
```diff
@@ -66,6 +66,20 @@ export const offlineToolsPatientsLink = getSyncLifecycle(
   options
 );
 
+export const offlineToolsActionsLink = getSyncLifecycle(
+  () =>
+    OfflineToolsNavLink({
+      page: "actions",
+      title: "Actions",
```
Comment (vasharma05) [inline packages/apps/esm-offline-tools-app/src/index.ts:73]: Please don't pass the title from here, instead put the title in the `OfflineToolsNavLink` with `t` from the `useTranslation` to allow translation for the **Actions** title.

File: packages/apps/esm-offline-tools-app/src/offline-actions/offline-actions-table.component.tsx (hunk 1 above)
Comment (icrc-jofrancisco) [reply packages/apps/esm-offline-tools-app/src/offline-actions/offline-actions-table.component.tsx]: Done. Thanks @vasharma05

File: packages/apps/esm-offline-tools-app/src/offline-patients/patient-name-table-cell.component.tsx (hunk 2 above)
Comment (icrc-jofrancisco) [reply packages/apps/esm-offline-tools-app/src/offline-patients/patient-name-table-cell.component.tsx]: Done. Thanks @vasharma05

File: packages/apps/esm-offline-tools-app/src/index.ts (hunk 3 above)
Comment (icrc-jofrancisco) [reply packages/apps/esm-offline-tools-app/src/index.ts]: Done. Thanks @vasharma05

**Review by ibacher — APPROVED** [review]
